design-reviews/readme.md

```diff
@@ -14,11 +14,37 @@
 - Continue to iterate on design after Game Plan review
 - Generate useful technical artifacts that can be referenced by Microsoft and customers
 
+## Measures
+
+### Cost of Change
+
+When incorporating design reviews as part of the engineering process, decisions are front-loaded before implementation begins. Making a decision of using Azure Kubernetes Service instead of App Services at the design phase likely only requires updating documentation. However, making this pivot after implmentation has started or after a solution is in use is much more costly.
+
+Are these changes occuring before or after implementation and how large of effort are they typically.

```

Comment:
These can be re-written as multiple points.
Are these changes occurring before or after implementation?
How large of effort are they typically?
- Also misspelled `occurring`.